### PR TITLE
Add success button

### LINKED ISSUE #1: Add success toast
After macro is completed, need to add well-looking success toast.
Step 1:
- [x] make simple toast - logic
* show it after macro completed
* show text - Sheets successfully merged!
- [x] design well-looking styled success toast
* background and form style
* button style - must be well look like native excel or similar

Button states: 
- on hover
- idle

Reference how to do that:
https://www.thespreadsheetguru.com/blog/build-modern-vba-userforms

Step 2:

- [x] Make danger toast
button: ~red
text: Something went wrong.. Please try again, or ask for help at GitHub.
- [x]  Add ErrorHanlder with danger toast. Danger toast must show error number and description.
- [x] Make info toast:
text: String and grey button

Step 3:
- [x] Study class (evening -> 11)
* How to initialize class
* How to write and call class method
* How to write and call class sub
* How to make parent class
* Study class private/public variables

Parent class = Vegetable
Private variable: _age
Method: age() return vegetable age 
Sub: grow(days) vegetable age + days

Successor class = Carrot, potato

- [ ] Write toast class
Methods / Subs:
1. show type: enum, text: string
2. hide

Types: Success / Danger/Info

How to make a class that can manage alerts?
1. So that the alert call code is in one place
2. Create two methods in the class, open and close the alert
3. One method to open is to contain the type of alert to open.
For example, info, danger, success and alert text. For types do enum
To select an alert inside the method, open either through If Else or Salect case "

Usage:
toast.show type, text
toast.hide

### PR DESCRIPTION
**Pull request by mixev**
Close #1

### CODE REVIEW

File: mSheetConcat.bas [hunk 1]
```diff
@@ -1,10 +1,12 @@
-Attribute VB_Name = "MainModule"
+Attribute VB_Name = "mSheetConcat"
 Option Explicit
 
 Private Function SheetExistence( _
   ByRef wbkActive As Workbook, _
   ByVal strSheetNameToFind As String, _
   ByVal blnSheetExists As Boolean) As Boolean
+  
+  Dim test as String
```
Comment (Arenukvern) [inline mSheetConcat.bas]:
```suggestion
  
```

File: mSheetConcat.bas [hunk 2]
```diff
@@ -64,6 +69,7 @@ Public Sub ActiveSheetsConcat()
   Set wbkActive = ActiveWorkbook
   ' We need to do check of an extension
   '
+  On Error GoTo ErrorHandler
```
Comment (Arenukvern) [inline mSheetConcat.bas]: Need to move to right after sub begins

File: mSheetConcat.bas [hunk 3]
```diff
@@ -46,6 +48,9 @@ End Function
 
 Public Sub ActiveSheetsConcat()
   Application.Calculation = xlManual
+  Dim dangerText As String
```
Comment (Arenukvern) [inline mSheetConcat.bas]: Move declaration to error handler, it will be more readable

**Review by Arenukvern — CHANGES_REQUESTED** [review]

File: mSheetConcat.bas (hunk 2 above)
Comment (mixev) [reply mSheetConcat.bas]: made fix position

File: mSheetConcat.bas (hunk 3 above)
Comment (mixev) [reply mSheetConcat.bas]: ready

Comment (Arenukvern) [reply mSheetConcat.bas]: @mixev  check it, I cannot see any changes 
Dim dangerText As String needs to be in error handler

**Review by Arenukvern — CHANGES_REQUESTED** [review]

**Review by Arenukvern — APPROVED** [review]
👍 💯

**Pull request merged**